docker-compose.yml

```diff
@@ -8,7 +8,9 @@ services:
         - http_proxy
         - https_proxy
         - no_proxy
-    image: decinosubmissionbackend/server:latest
+    image: tsed/server:latest
```

Comment:
this image name differs now, is it a conflict with the one in `package.json`?

Reply:
I think this ios right? dunno